### ISSUE TITLE
[CSS] Add underlines to body text links

### ISSUE THREAD

**Issue by sboots** [ytgov/ckanext-yukondesign#113]
For accessibility reasons, links should be underlined throughout the site. (Like #112 , I'll update the Bootstrap template as soon as I have a chance!).

The only exceptions (that do **not** need to be underlined) are,
- the Français (or English) language switcher link
- the current page, in a breadcrumb list (in grey instead of green)
- button-styled links (like the pagination buttons, or the admin section header buttons, or the "manage" or "download" buttons etc.)
- resource heading (title) links
- any CKAN interface components (like the top menu etc.) that aren't affected by the current theme

We can see if these look better with or without underlines:
- search result heading links
- search page filter links in the sidebar

**Comment by sboots** [comment]
You can see an example of how this is implemented in the Bootstrap SCSS repository,
https://open-data-static-files.service.yukon.ca/yukon-bootstrap5-sass-js/#links
